courses/management/commands/sync_grades_and_certificates.py

```diff
@@ -89,46 +91,53 @@ def handle(self, *args, **options):  # pylint: disable=too-many-locals
         results = []
         for edx_grade, user in edx_grade_user_iter:
             result = []
-            course_run_grade, created, updated = ensure_course_run_grade(
+            course_run_grade, created_grade, updated_grade = ensure_course_run_grade(
                 user=user,
                 course_run=run,
                 edx_grade=edx_grade,
                 should_update=should_update,
             )
 
-            if created:
-                result.append("New grade record created")
-            elif updated:
-                result.append("Grade record updated with new data from edX grades API")
-
             if override_grade is not None:
                 course_run_grade.grade = override_grade
                 course_run_grade.passed = bool(override_grade)
                 course_run_grade.letter_grade = None
                 course_run_grade.set_by_admin = True
                 course_run_grade.save_and_log(None)
-                result.append(
-                    "Grade overriden - new grade value: {}".format(
-                        course_run_grade.grade
-                    )
-                )
+                result.append("grade value override: {}".format(course_run_grade.grade))
```

Comment:
The `result` variable and this line can be removed

Reply:
Ah, missed that one